torchci/pages/kpis.tsx

```diff
@@ -25,7 +26,83 @@ export default function Kpis() {
     
     return (
         <Grid container spacing={2}>
-            <Grid item xs={true} height={ROW_HEIGHT}>
+            <Grid item xs={6} height={ROW_HEIGHT}>
+                <TimeSeriesPanel
+                title={"Avg Time To Signal - E2E (Weekly)"}
+                queryName={"time_to_signal"}
+                queryCollection={"pytorch_dev_infra_kpis"}
+                queryParams={[
+                    {
+                        name: "buildOrAll",
+                        type: "string",
+                        value: "all",
+                    },
+                    ...timeParams,
+                ]}
+                granularity={"week"}
+                timeFieldName={"week_bucket"}
+                yAxisFieldName={"avg_tts"}
+                yAxisLabel={"Hours"}
+                yAxisRenderer={(unit) => {
+                    return `${unit}`;
```

Comment:
nit: You can do something like `unit => `${unit}`. You don't need the curly braces here unless you wanted to do more logic other than return that.

Also since we're using this function in multiple places, you can do something like 
```
const unitRenderer(unit) => {
return `${unit}`
}

yAxisRenderer = {unitRenderer} 

```

Reply:
good point